shared/common/common.go

```diff
@@ -42,14 +43,21 @@ func CreateDirWithFileMode(dirPath string, perm os.FileMode) error {
 	return nil
 }
 
+// ExecCommand run the given command and returns the output
+func ExecCommand(cmd string, args ...string) (string, string, error) {
```

Comment:
This can now be replaced with the `cmd` lib we have in `pkg/test` - https://github.com/knative/pkg/blob/master/test/cmd/command.go, if you like.

Reply:
Thanks. Done.